src/main/kotlin/com/openlattice/shuttle/Shuttle.kt

```diff
@@ -245,6 +293,15 @@ fun main(args: Array<String>) {
     }
 }
 
+fun makeAWSS3Client( region: String ): AmazonS3 {
+    return AmazonS3ClientBuilder
+            .standard()
+            .withPathStyleAccessEnabled(true)
+            .withRegion( region )
+            .withCredentials(AWSStaticCredentialsProvider(AnonymousAWSCredentials()))
```

Comment:
Will you make sure this won't prevent IAM service roles assigned to EC2 instances from being used? My instinct is that credentials should be explicitly provided or service roles to make sure people don't do bad things with buckets.

Reply:
Whoops, think that might have been a goof. I'm pretty sure `DefaultAWSCredentialsProviderChain` will do the right thing